src/app/contact/Form.tsx

```diff
@@ -49,6 +66,13 @@ export default function Form({ className }: { className?: string }) {
                         Submit
                     </Button>
                 </div>
+                {submitInfo && (
+                    <div
+                        className={`col-span-2 text-center ${isSuccess ? 'text-green-500' : 'text-red-500'}`}
```

Comment:
should use library like `tw-merge` for this but since we didn't install, it's fine.